Flow.Launcher/SettingWindow.xaml.cs

```diff
@@ -263,7 +263,7 @@ private async void OnCheckUpdates(object sender, RoutedEventArgs e)
 
         private void OnRequestNavigate(object sender, RequestNavigateEventArgs e)
         {
-            SearchWeb.NewBrowserWindow(e.Uri.AbsoluteUri);
+            SearchWeb.NewTabInBrowser(e.Uri.AbsoluteUri);
```

Comment:
Can you also replace the use of `SearchWeb.NewBrowserWindow` in other places in settings?

Reply:
Sure.